### PR TITLE
data:Yushi5058

### PR DESCRIPTION
**Pull request by Yushi5058**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4739"></a>

### CODE REVIEW

File: data/Yushi5058.json [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
+	],
+	"links": [
 		{
-			"name": "Github",
+			"name": "Follow me on Github",
 			"url": "https://github.com/Yushi5058",
 			"icon": "FaGithub"
-		}
-	]
+	],
```
Comment (kumarsonsoff3) [inline data/Yushi5058.json:17]: There is a missing bracket `}` & extra comma `,` which is leading the `json` check fail.

```suggestion
	}
     ]
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: data/Yushi5058.json (hunk 1 above)
Comment (Yushi5058) [reply data/Yushi5058.json:17]: Thank you for pointing that for me. I'll resolve it :)

File: data/Yushi5058.json [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
-	"bio": "Self-taught web developer, passionate about the tech industry and constantly improving my skills to advance my career.",
-	"links": [{
-			"name": "Discord",
-			"url": "discord.com/users/1071758711330193478",
-			"icon": "FaDiscord"
-		},
+	"bio": "Self taught web dev | Front-end web dev | Linux & Bash Enthusiast",
+	"tags": ["Open Source", "Front-End", "Linux", "Javascript"],
+	"socials": [
+		{"icon": "FaDiscord", "url":"discord.com/users/1071758711330193478"},
```
Comment (krshkun) [inline data/Yushi5058.json:7]:
```suggestion
```

This will result in 404, discord does not support this feature yet

**Review by krshkun — CHANGES_REQUESTED** [review]

**Review by krshkun — APPROVED** [review]
Looks good

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**